src/components/network/constants.js

```diff
@@ -32,6 +32,12 @@ export const UPDATE_TYPE = [
     'deletingInProgress',
 ];
 
+export const REGULATION_MODES = [
+    { id: 'FIXED_TAP', label: 'FixedTap' },
+    { id: 'CURRENT_LIMITER', label: 'CurrentLimiter' },
+    { id: 'ACTIVE_POWER_CONTROL', label: 'ActivePowerControl' },
+];
```

Comment:
Maybe use an object instead of a table, like this ?
`export const REGULATION_MODES = {`
`FIXED_TAP: { id: 'FIXED_TAP', label: 'FixedTap' },`
`CURRENT_LIMITER: { id: 'CURRENT_LIMITER', label: 'CurrentLimiter' },`
`ACTIVE_POWER_CONTROL: { id: 'ACTIVE_POWER_CONTROL', label: 'ActivePowerControl' },`
`}`

This way, you can use `REGULATION_MODES.ACTIVE_POWER_CONTROL.id` instead of `REGULATION_MODES[2].id`